internal/api/front/perses/src/app/project/dashboard/dashboardfeed.model.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021 Amadeus s.a.s
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export interface Serie {
+  metric: Record<string, string>;
+  values: [number, string][];
+}
+
+export interface Result {
+  result: Serie[];
```

Comment:
If something goes wrong when contacting Prometheus, it will generate an error that would appear in this interface which is matching the golang struct : https://github.com/perses/perses/blob/93a0c7961d682e78a691f83c51561257eb21afaa/pkg/model/api/v1/dashboard_feed.go#L23-L26

Reply:
So you need to add the attribute `err` like that: 

```typescript
export interface Result {
  result: Serie[];
  err?: string;
}
```